.gitignore

```
@@ -2,6 +2,8 @@
 
 # dependencies
 /node_modules
+.package-lock.json
+#Added this to avoid changes to Dependency files. 
```

```suggestion
#Added this to avoid changes to Dependency files. 
.package-lock.json
```

for me the comments come before the line of code but this is only my personal preference